modules/bullet/godot_result_callbacks.cpp

```diff
@@ -333,14 +333,6 @@ void GodotDeepPenetrationContactResultCallback::addContactPoint(const btVector3
 		m_other_compound_shape_index = isSwapped ? m_index0 : m_index1;
 		m_pointWorld = isSwapped ? (pointInWorldOnB + (normalOnBInWorld * depth)) : pointInWorldOnB;
 
-		const btCollisionObjectWrapper *bw0 = m_body0Wrap;
-		if (isSwapped)
-			bw0 = m_body1Wrap;
-
-		if (bw0->getCollisionShape()->getShapeType() == CUSTOM_CONVEX_SHAPE_TYPE) {
-			m_pointNormalWorld = bw0->m_worldTransform.getBasis().transpose() * btVector3(0, 0, 1);
```

Comment:
Have you actually tested this? Because the sample project in the issue works for me identically in both GodotPhysics and Bullet, with both "Slips On Slope" set to on and off.

IIRC this code I removed practically breaks "Slips On Slope" set to on, so that's why I removed it.

Reply:
Yes I did, and the idea is that when slips on slope is true this condition should fail. I can give it a try later

Reply:
Ok so this one is correct and you can leave this as is now